roles/logging/tasks/logstash.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: uninstall filebeat(ubuntu)
```

Comment:
given is same package name, have you tried out ansible package module? Using Package you don;t need task per os to uninstall a package.